indra/newview/llviewerstats.cpp

```diff
@@ -274,6 +274,16 @@ void LLViewerStats::updateFrameStats(const F64Seconds time_diff)
         // old stats that were never really used
         F64Seconds jit = (F64Seconds)std::fabs((mLastTimeDiff - time_diff));
         sample(LLStatViewer::FRAMETIME_JITTER, jit);
+
+        if (gFocusMgr.getAppHasFocus())
+        {
+            mForegroundFrameStats.push(F32(F64(time_diff)));
```

Comment:
The double casting looks odd, but I assume you tried it with simpler code.

Reply:
Thought I had but it turns out the simpler formulation still works.